File: ncDados/dEstadoCivil.vb
Namespace nsDados

  Public Class ColecaoEstadoCivil
    Inherits List(Of dEstadoCivil)
  End Class

  Public Class dEstadoCivil

    Private _codigo As String
    Private _descricao As String

    Public Property codigo() As String
      Get
        Return _codigo
      End Get
      Set(ByVal value As String)
        _codigo = value
      End Set
    End Property

    Public Property descricao() As String
      Get
        Return _descricao
      End Get
      Set(ByVal value As String)
        _descricao = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dEstadoCivil.cs
using System;
using System.Collections.Generic;

namespace nsModelos
{
    public class ColecaoEstadoCivil : List<dEstadoCivil>
    {
    }

    public class dEstadoCivil
    {
        private string _codigo;
        private string _descricao;

        public string codigo
        {
            get => _codigo;
            set => _codigo = value;
        }

        public string descricao
        {
            get => _descricao;
            set => _descricao = value;
        }
    }
}
